protobuf/util_msg.proto

```diff
@@ -0,0 +1,22 @@
+syntax="proto3";
```

Comment:
We may drop `_msg` in the filename. As the file extensions are always either `.proto`, `.pb.h`, etc., it is clear that the file is protobuf stuff. Same goes for `bess_msg.proto`, `module_msg.proto`, and `port_msg.proto`.